<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QDialog" name="DownloadDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DownloadDialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="bottomLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>410</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="controlButtonsHorizontalLayout">
    <item>
     <spacer name="controlHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okPushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>392</width>
     <height>388</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="downloadFolderGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Download Folder</string>
      </property>
      <widget class="QWidget" name="topLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>371</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="downloadHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="downloadFodlerLineEdit">
          <property name="text">
           <string>e:/test/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browsePushButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="torrentContentsGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>330</height>
       </size>
      </property>
      <property name="title">
       <string>Torrent Contents</string>
      </property>
      <widget class="QWidget" name="centerLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>371</width>
         <height>297</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="torrentContentsVerticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QFormLayout" name="torrentInfoFormLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="torrentNameInfoVerticalLayout">
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="commentLabel">
              <property name="text">
               <string>Comment:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dateLabel">
              <property name="text">
               <string>Date:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="torrentInfoVeticalLayout">
            <item>
             <widget class="QLabel" name="torrentNameLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="torrentCommentLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="torrentDateLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="downloadListTreeWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="torrentHorizontalLayout">
          <item>
           <spacer name="torrentHorizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="torrentSizeLabel">
            <property name="text">
             <string>2 files, 2.17 GB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
